### ISSUE TITLE
Cannot read property 'pubdate' of undefined (ssg/dist/coreData.js:80:35)

### ISSUE THREAD

**Issue by louwers** [sw-yx/ssg#12]
After `yarn add ssg && touch ssg.config.js`:

```
[bart@karuna ssg]$ yarn ssg dev
yarn run v1.21.1
warning package.json: No license field
$ /home/bart/src/ssg/node_modules/.bin/ssg dev
> Cannot read property 'pubdate' of undefined
TypeError: Cannot read property 'pubdate' of undefined
    at /home/bart/src/ssg/node_modules/ssg/dist/coreData.js:80:35
    at Array.sort (<anonymous>)
    at Object.<anonymous> (/home/bart/src/ssg/node_modules/ssg/dist/coreData.js:79:18)
    at Generator.next (<anonymous>)
    at fulfilled (/home/bart/src/ssg/node_modules/ssg/dist/coreData.js:5:58)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by swyxio** [comment]
thanks for reporting! will check on this

**Comment by swyxio** [comment]
i've fixed this with v0.0.58 :) thanks for the report.
